config/environment.js

```diff
@@ -2,5 +2,11 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    moment: {
+      // To cherry-pick specific locale support into your application.
+      // Full list of locales: https://github.com/moment/moment/tree/2.10.3/locale
+      includeLocales: ['en', 'fr']
```

Comment:
But if it does not impact the library, you don’t need to modify the config? You modified it just to test that it was rendering the right names in french?